src/main/java/com/exasol/sql/ValueTable.java

```diff
@@ -1,6 +1,8 @@
 package com.exasol.sql;
 
-import java.util.*;
+import java.util.ArrayList;
```

Comment:
I am going to check the formatter settings again. Please correct by hand until then.

In the `org.eclipse.jdt.ui.prefs` I see:

```
org.eclipse.jdt.ui.ondemandthreshold=3
org.eclipse.jdt.ui.staticondemandthreshold=3
```

Both are correct.

That means the problem must be on the IntelliJ side.

Reply:
fixed